cypress/e2e/certifications.cy.js

```diff
@@ -0,0 +1,17 @@
+describe.only('Certifications', () => {
+  describe('RGE', () => {
+    it('Should display two certifications', () => {
+      cy.visit('/labels-certifications/528163777');
+      cy.get('.full-table > tbody').find('tr').its('length').should('eq', 2);
```

Comment:
This test is very likely to break as one might add certificates or remove them. I would rather add a test on section being present or displaying the not found message

Reply:
I just removed this one